docs/src/markdown-pages/aptitudes/filesystem.md

```diff
@@ -1,17 +1,31 @@
 ---
-name: "Filesystem"
-links_js: "Filesystem"
-description: "Provides read and write access to the filesystem, enabling Loop Authors to read and write files."
+name: 'Filesystem'
+links_js: 'Filesystem'
+description: 'Provides read and write access to the filesystem, enabling Loop Authors to read and write files.'
 ---
+
 Provides read and write access to the filesystem, enabling Loop Authors to read and write files.
 
-[Permissions must be declared](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript#loop-permissions) for a Loop to use the Filesystem Aptitude. 
+[Permissions must be declared](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript#loop-permissions) for a Loop to use the Filesystem Aptitude.
 
 The filesystem permissions object must be populated with the locations you want to access in the following format:
+
 ```
 ldk:
  permissions:
    filesystem:
      pathGlobs:
        -  value: '/tmp/*.txt'
-```
\ No newline at end of file
+```
+
+### Filesystem Notes
```

Comment:
Are we allowing loop authors to use users directory like: `/Users/username/Desktop/Test.xlsx`?
If so do we want to add a blurb about it in documentation? If not then I think we may need to talk about how loop authors will persist data between loop runs. But this is probably another conversation.

Reply:
Added a new section pertaining to absolute paths.